assets/core/csr_approver_clusterrole.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
shouldn't these assets be vendored (and possibly symlinked if you need them at this location)?

Reply:
This manifest comes from https://github.com/openshift/cluster-kube-controller-manager-operator/blob/e827c030ba3cd632668820cca73dfcbc986cc5a2/bindata/assets/kube-controller-manager/namespace-openshift-infra.yaml, and Microshift doesn't include operators, so it doesn't have a module dependency on kcm-operator (and isn't present in under vendor).

Reply:
We have yaml manifests that are auto-imported from our rebase script (so if updates happen to the yamls, the yamls here will get updated). This would need to be one of those as far as I understand.